File: src/main/java/yuu/rpg/Util/ActionBar.java
package yuu.rpg.Util;

import net.md_5.bungee.api.ChatMessageType;
import net.md_5.bungee.api.chat.TextComponent;
import org.bukkit.entity.Player;

public class ActionBar {

    public static void sendActionBar(Player p, String message) {
        p.spigot().sendMessage(ChatMessageType.ACTION_BAR, new TextComponent(message));
    }
}

File: src/main/java/yuu/rpg/ActionBar.kt
package yuu.rpg

import net.md_5.bungee.api.ChatMessageType
import net.md_5.bungee.api.chat.TextComponent
import org.bukkit.entity.Player

object ActionBar {

    fun sendActionBar(p: Player, message: String) {
        p.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent(message))
    }

    fun statusActionBar(p: Player, HP: String, MP: String) {
        p.spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent("HP:" + HP + "MP:" + MP))
    }
}
